src/shadowbox/server/managed_access_key.ts

```diff
@@ -165,11 +150,11 @@ class ManagedAccessKeyRepository implements AccessKeyRepository {
       try {
         this.saveConfig();
       } catch (error) {
-        return Promise.reject(new Error(`Failed to save config: ${error}`));
+        throw new Error(`Failed to save config: ${error}`);
```

Comment:
While you're in here, how about logging just `error.message`?

Also, `saveConfig` is a one liner which is only used by this function - I would just fold it in here.

Reply:
The error is logged by the caller. I think it works better if an error is only logged by whoever handles it. Otherwise you end up with inconsistent and duplicate logging.

Reply:
`saveConfig` is also called from `removeAccessKey`

Reply:
Sorry, I missed the other calls. For logging, I just meant change:
```ts
throw new Error(`Failed to save config: ${error}`);
```
to:
```ts
throw new Error(`Failed to save config: ${error.message}`);
```

Reply:
Oh, this error has a proper `toString` method? I may have had my head buried in weird Windows client code for too long.